SEP-0002-revised.md

```diff
@@ -0,0 +1,262 @@
+* The Advisory Board may appoint members to its number up to the maximum number of members.
+* The Advisory Board should work to ensure that the composition of the Advisory Board represents a diverse cross-section of the scientific community with an interest in SunPy. This diversity should consider things such as research topic, career stage, as well as all other forms of diversity such as gender.
+* An Advisory Board member may be removed by a two-thirds majority vote of the Advisory Board or by Project Members following the same procedure as for the removal of a member of the Steering Committee.
+
+#### Meetings
+
+* The Advisory Board shall meet with the Steering Committee at least every quarter.
+* At least one week (7 days) notice of a meeting shall be provided to all eligible to attend.
+* Notice of meetings shall provide the information necessary to participate in the meeting.
+* Minutes for the meetings shall be recorded and made public as much as is appropriate.
+
+The Advisory Board shall organize itself as it sees fit.
+
+#### Advise and Consent
+
+The Steering Committee shall present the state of the project and future plans at meetings of the Advisory Board on a yearly basis.
+The Advisory Board shall advise and consent on those plans.
+
+#### Requiring a Project Vote
+
+In the unlikely event that the Advisory Board does not agree with a decision made by the Steering Committee, it can require Project Members to consent to the decision through a vote (see below for process).
+
+### The Steering Committee
+
+The role of the Steering Committee, which is chaired by the Lead Developer, is to lead the day-to-day operations of the project.
+The Steering Committee shall be responsible for the running of the project under advisement of the Advisory Board.
+This includes:
+
+* holding regular meetings with the developer community (at least monthly)
+* making regular reports to the Advisory Board (at least quarterly)
+* having ownership privileges to the SunPy GitHub organization and other community assets
+* creating, maintaining and enforcing the SunPy code of conduct (see below)
+* accepting or rejecting SEPs based on community consensus (excluding this SEP, see below)
+* delegating responsibility and authority to other community roles
+* planning and managing finances and financial processes of the project.
+
+#### Composition
+
+The Steering Committee is formed of three members of the SunPy community with equal status on the committee, and they do not need to be Project Members.
+The members of the committee are elected into three roles to help with distributing the work load, these three roles are:
+
+* **The lead developer** chairs the Steering Committee and is responsible for holding regular (at least monthly) meetings, and ensuring that the committee is represented at all Advisory Board meetings.
+* **The deputy lead developer** in the absence of the lead developer, the deputy shall perform all the duties of the lead developer.
```

Comment:
Thinking about this some more, I worry more that this would make the lead developer dominate the steering committee since that position owns 2/3 of the votes.

Reply:
> that position owns 2/3 of the votes

eh? how'd you figure that?

Reply:
Perhaps the best thing is to just come up with a different role, or even get rid of the roles completely?

Reply:
Would we then remove the lead developer role completely then? Not sure I like that. The steering committee itself could be considered all the lead developers  but I don't think I like that either. We could let the steering committee choose one of its own members to be lead developer but that would remove the choice from the project team members. If we want to keep the lead developer role I would still prefer we decide on two other equally important roles for the steering committee members. Financial officer feels just as important. We just need one more. That way the steering committee is composed of three equally important and involved people. A triumvirate!

Reply:
> That way the steering committee is composed of three equally important and involved people. A triumvirate!

To be really clear it is at the moment, everyone on the committee has an equal vote on all matters. The roles are really briefs rather than anything else.

I am up for suggestions of a third role, but I can't think of one.

Reply:
Numpy doesn't have any titles for their Steering Council
Scipy has a BDFL and Steering Council with the only named role as a Council Chair.

Reply:
@ehsteve Says [elsewhere](https://github.com/sunpy/sunpy-SEP/pull/62#discussion_r1146590188):

> Again harping on the deputy position on the Steering Committee, it's weird that the Project Members should the lead developers' deputy. The lead developer may want to choose their own deputy and the third Steering Committee member should have their own role. How about "Chief Diversity Officer"?

Reply:
We discussed this at the community meeting even though there wasn't a strong feelings (and none of us really had a problem with the deputy role) we thought that it would be better to have the steering committee divide the responsibilities between themselves as they see fit. Therefore we propose that we remove the named roles from the document and make it clear that all the responsibilities of the committee are equally shared between all members.

This also has the side effect of making the elections simpler as you are no longer electing people into a specific role, just 3 people onto the steering committee.

Reply:
Hmm.  I'd still prefer to have a single person identified as "lead developer".  Perhaps the steering committee can be given the authority to self-assign titles to each of its members, with "lead developer" being a title that must be used.  In this scheme, the "lead developer" would not have any greater influence than any other member of the steering committee: the actual responsibilities could still be divided as the committee sees fit.

> This also has the side effect of making the elections simpler as you are no longer electing people into a specific role, just 3 people onto the steering committee.

The downside of this simplification is that we would have to be disinclined to vote for "specialists" lest the skills of the steering committee become imbalanced.  As an extreme example, you could accept having one member who's awful at running regular meetings if that person is excellent at managing finances, but you wouldn't want to end up with a steering committee of three members like that.

Reply:
> I'd still prefer to have a single person identified as "lead developer"

I leant to this as well, which is why I proposed what I did to start off with. Can you elaborate as to why?

> Perhaps the steering committee can be given the authority to self-assign titles to each of its members, with "lead developer" being a title that must be used.

Yeah, I think this works, I can come up with some verbiage to make it clear the committee can give people titles if they want to divide the work up and that they should nominate a "lead developer" to chair the committee?

> The downside of this simplification is that we would have to be disinclined to vote for "specialists" lest the skills of the steering committee become imbalanced.

Yeah, I think that for now this is unlikely to be a practical problem, if we see it becoming an issue we could always modify the process to fix it. For now I think the win of less complex admin beats the hypothetical problems?